Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -349,3 +349,13 @@
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpsuit/prosecutorred.rsi
 
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformBlackTurtleneck
+  name: black turtleneck
+  description: A simple black turtleneck with the sleeves cut off.
```

Comment:
this should use the rmc thing for taking off sleeves and be sleeved by default

Reply:
Alright, hoping I didn't break anything by merging from the master branch, but it should be good.